webpack.config.js

```diff
@@ -125,13 +125,13 @@ const config = (module.exports = {
     },
   },
 
+  optimization: {
+    splitChunks: {
+      chunks: "all",
+    },
+  },
+
```

Comment:
# Replace `CommonsChunkPlugin` with `splitCunks`
```
[cljs] shadow-cljs - server starting ...[webpack-cli] Failed to load '/Users/offbynone/code/work/metabase/webpack.config.js' config
[frontend] [webpack-cli] TypeError: webpack.NamedModulesPlugin is not a constructor
[frontend]     at Object.<anonymous> (/Users/offbynone/code/work/metabase/webpack.config.js:265:5)
[frontend]     at Module._compile (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
[frontend]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
[frontend]     at Module.load (internal/modules/cjs/loader.js:928:32)
[frontend]     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
[frontend]     at Module.require (internal/modules/cjs/loader.js:952:19)
[frontend]     at require (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
[frontend]     at loadConfig (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1327:31)
[frontend]     at WebpackCLI.resolveConfig (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1459:44)
[frontend]     at WebpackCLI.createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1837:33)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/@webpack-cli/serve/lib/index.js:81:40)
[frontend]     at Command.listener [as _actionHandler] (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:922:31)
[frontend]     at Command._parseCommand (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1503:14)
[frontend]     at Command._dispatchSubcommand (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1443:18)
[frontend]     at Command._parseCommand (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1460:12)
[frontend]     at Command.parse (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1292:10)
error Command failed with exit code 2.
```

Searching online for a solution:
- https://stackoverflow.com/questions/67381359/typeerror-webpack-namedmodulesplugin-is-not-a-constructor
- https://www.gitmemory.com/issue/webpack/webpack/11637/706718119
> This plugin was removed, you even don’t need this anymore, we use better logic for long term caching out of box